components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -203,6 +203,7 @@ class GeckoEngine(
         return installWebExtension(ext, onSuccess, onError)
     }
 
+    @Suppress("Deprecation")
```

Comment:
See below. Let's put this on the `register` call with a link to https://github.com/mozilla-mobile/android-components/issues/6356